delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/projects/ProjectsImpl.scala

```diff
@@ -33,15 +34,29 @@ final class ProjectsImpl private (
   override def create(
       ref: ProjectRef,
       fields: ProjectFields
-  )(implicit caller: Subject): IO[ProjectResource] = {
+  )(implicit caller: Subject): IO[ProjectResource] =
     for {
       resource <- eval(CreateProject(ref, fields, caller)).span("createProject")
       _        <- scopeInitializations
-                    .parUnorderedTraverse(_.onProjectCreation(resource.value, caller))
+                    .parUnorderedTraverse { init =>
+                      init
+                        .onProjectCreation(resource.value, caller)
+                        .attemptNarrow[ScopeInitializationFailed]
+                        .flatMap {
+                          case Left(e)  =>
+                            errorStore
+                              .save(init.entityType, resource.value.ref, e)
+                              .onError(e =>
+                                logger.error(e)(
+                                  s"Failed to save error for '${init.entityType}' initialization step on project '${resource.value.ref}'"
+                                )
+                              )
+                          case Right(_) => IO.unit
+                        }
```

Comment:
There will be something along the lines of `onError { case e: ScopeInitializationFailed => ???}` you could use

Reply:
You're right, I already had the same issue at some point; I keep forgetting about `recover`